### ISSUE TITLE
what is invert_imu

### ISSUE THREAD

**Issue by HaoyunXu** [koide3/hdl_localization#11]
On default, the invert_imu is enabled but the localization just failed when the car turns. Then I turned it off, and everything works perfectly find. Could you give me a hint of what this invert_imu does?

**Comment by koide3** [comment]
Hi @HaoyunXu ,
If it is enabled, all IMU data will be inverted (multiplied by -1). It was made for backward compatibility with a specific sensor. I think I will make it disable by default. Thanks for reporting.

**Comment by narutojxl** [comment]
I did a test, it seems that when using [hdl_400.bag.tar.gz](http://www.aisl.cs.tut.ac.jp/databases/hdl_graph_slam/hdl_400.bag.tar.gz) with imu, the imu's acc and gyro in the bag, we should enable `invert_imu_acc` and `invert_imu_gyro`. 
Note: 
imu's orientation is not used.  
Before using imu, we should firstly transform imu's data into laser, if they is not aligned. Am i correct @koide3 :)?

**Comment by koide3** [comment]
Yes, the acc and gyro in the bag are inverted, and thus ```invert_imu_acc``` and ```invert_imu_gyro``` need to be enables. ```plot_status.py``` is useful to validate the IMU axes directions that shows prediction errors with/without IMU. If the errors with IMU are smaller than without IMU, your IMU setting should be fine.

![Screenshot_20210122_114421](https://user-images.githubusercontent.com/31344317/109448474-65aee100-7a89-11eb-97fe-a4fffb812f82.png)

In case the LIDAR and IMU have different coordinate systems, it's better to change ```odom_child_frame_id``` to the imu frame. Then, input point cloud is transformed in the IMU frame, and localization should be done without problems.

**Comment by narutojxl** [comment]
Thanks for koide3's detailed reply. :+1:

**Comment by DCYzhao** [comment]
![Image](https://github.com/user-attachments/assets/7d18a915-ba96-4f2b-852c-2e469bf5fca3)
When using IMU for prediction, the translation prediction is not ideal. May I ask what the issue is, and is this normal?
